content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Integer/2. Uint.md

```diff
@@ -0,0 +1,41 @@
+---
+title: 2. Uint
+taxonomie:
+  - ib-19.2.Datatype-Primitief-Uint.OI
+  - ib-19.3.Datatype-Primitief-Uint.OI
+---
+
+### Wat is een Uint 
+Een Uint, ofwel unsigned integer, is een datatype die dezelfde ruimte in beslag neemt als de [[content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Integer/1. Integer|Integer]]. De Uint kan alleen geen negatieve getallen opslaan, maar gebruikt deze ruimte om een twee keer zo groot positief getal op te kunnen slaan (4,294,967,295). 
+
+Voor de rest werkt de Uint hetzelfde als de [[content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Integer/1. Integer|Integer]].
+
+> [!TIP] Casus
+> Stel je hebt een evenement met waar van tevoren tickets voor worden gekocht. Je wilt bijhouden hoeveel tickets er zijn verkocht.  
+
+### Hoe zit een Uint in elkaar
+De structuur van de Uint werkt hetzelfde als die van de [[content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Integer/1. Integer|Integer]]. Alleen moet je nu Uint voor het variable neerzetten.
```

Comment:
Opmerking: het codevoorbeeld gebruikt geen hoofdletter. Handig om één lijn te trekken?

Reply:
Ik denk dat je hier bedoelt dat in de code 'uint' moet staan, maar dat in de tekst 'Uint' staat. Ik ben het er mee eens dat dit verwarrend zou kunnen zijn en had dit eerder ook al naar voren geschoven naar @martijnschuman en @TeunvdKleij. Omdat alle onderwerpen in de teksten met een hoofdletter worden geschreven, laat ik het nu zo staan. Volgensmij was het ook de bedoeling dat dat zo bleef, maar dat weten hun beter:).